fixtures/build.js

```diff
@@ -22,28 +20,32 @@ window.EXAMPLE_BUILD = {
       }
     },
     "strategy": {
-      "type": "STI",
-      "stiStrategy": {
-        "builderImage": "openshift/ruby-20-centos7:latest",
-        "image": "openshift/ruby-20-centos7:latest"
+      "type": "Source",
+      "sourceStrategy": {
+        "from": {
+          "kind": "DockerImage",
+          "name": "openshift/ruby-20-centos7:latest"
```

Comment:
so after all that, if this is really building from a `DockerImage`, `openshift/ruby-20-centos7:latest` is perfectly valid